maintainers/maintainer-list.nix

```diff
@@ -3951,6 +3951,12 @@
     githubId = 6430643;
     name = "Henry Till";
   };
+  henson = {
```

Comment:
additions to maintainer-list.nix should be separate commits

Reply:
thank you, I split it into a separate commit